external-applications/quarkus-workshop-super-heroes/src/test/java/io/quarkus/ts/openshift/heroes/workshop/HeroesOpenShiftIT.java

```diff
@@ -0,0 +1,193 @@
+package io.quarkus.ts.openshift.heroes.workshop;
+
+import io.quarkus.ts.openshift.common.AdditionalResources;
+import io.quarkus.ts.openshift.common.ManualDeployment;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import java.net.URL;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static javax.ws.rs.core.HttpHeaders.ACCEPT;
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.CREATED;
+import static javax.ws.rs.core.Response.Status.NO_CONTENT;
+import static javax.ws.rs.core.Response.Status.OK;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+@ManualDeployment(appName = "quarkus-workshop-hero", httpRoot = "/", knownEndpoint = "/")
+@AdditionalResources("classpath:openjdk-11-rhel7.yaml")
+@AdditionalResources("classpath:heroes-database.yaml")
+@AdditionalResources("classpath:hero.yaml")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class HeroesOpenShiftIT {
+
+    @TestResource
+    private URL url;
```

Comment:
This should IMHO be below the `static` fields. The constants should be first, then the `heroId` (where we probably should add a comment that it's a workaround for passing data between test methods; alternatively, we could make `heroId` an instance field and make the test `@TestInstance(Lifecycle.PER_CLASS)`), and then this instance field.